bin/epispot

```diff
@@ -8,7 +8,12 @@ except:
     print("Please install Fire to use epispot's CLI")
     print('>>> pip install fire')
 
+
 class bcolors:
```

Comment:
Can we remove the `bcolors` class and move all its variables to a new file `bcolors.py` and then 
```python
import bcolors
```

Reply:
We could but the cli is only in the epispot file, so it doesn't make sense.

Reply:
It is extremely important because `bin/epispot` is only the base CLI invocation; actual code will be imported from other files. Therefore, we need a separate file where the color codes are stored.

Reply:
This was fixed in commit https://github.com/epispot/epispot/pull/40/commits/a91fa0fbdc1198e994e13f033926d808a66bcb14